Product models (#72)
akeneo/onboarder-helpcenter: content/md/retailer-onboarder/define-suppliers-scope/define-product-requirements.md

:::

::: warning

> Custom attributes are not supported by the Onboarder and suppliers are not able to enrich data of custom attributes.

Reviewer: ```suggestion
Custom attributes are not supported by the Onboarder and suppliers are not able to enrich custom attributes data.
```

:::

::: warning

> When you [create a supplier](/onboarder/articles/create-supplier.html), you can chose to allow the product creation.

Reviewer: ```suggestion
When you [create a supplier](/onboarder/articles/create-supplier.html), you can choose to allow them to create products.
```
